src/index.html

```diff
@@ -20,8 +20,372 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="/src/images/logo.png"
```

Comment:
The alt attribute for the logo image is 'Moyo Logo'. Ensure that this text is descriptive enough for users who rely on screen readers. If 'Moyo Logo' is the best description, then it's fine, but consider if more context is needed.